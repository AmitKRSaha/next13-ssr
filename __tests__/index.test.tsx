import Home from "../pages/index";
import renderer from 'react-test-renderer';

describe('test', () => {

    it("test",() => {
        expect(true).toBeTruthy();
    })

    it('renders correctly', () => {

        const props ={
            repo : {},
            photos: {}
        }

        const tree = renderer
            .create(<Home />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})