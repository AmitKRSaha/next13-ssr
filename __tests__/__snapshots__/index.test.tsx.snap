// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders correctly 1`] = `
<main
  className="main className"
>
  Name 
  <span>
    test
  </span>
  <br />
  <button
    onClick={[Function]}
  >
    Change Name
  </button>
  <div>
    SSR
  </div>
  <div />
  <div>
    <div>
      <p
        className="italic"
      />
      <p />
    </div>
  </div>
</main>
`;
